[2023-12-10 19:32:39,387] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-10-10T00:00:00+00:00 [queued]>
[2023-12-10 19:32:39,728] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-10-10T00:00:00+00:00 [queued]>
[2023-12-10 19:32:39,749] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:32:39,764] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:32:39,778] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:32:39,991] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2022-10-10 00:00:00+00:00
[2023-12-10 19:32:40,007] {standard_task_runner.py:52} INFO - Started process 904 to run task
[2023-12-10 19:32:40,121] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2022-10-10T00:00:00+00:00', '--job-id', '214', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmplrh8cupy', '--error-file', '/tmp/tmper17ohz6']
[2023-12-10 19:32:40,138] {standard_task_runner.py:80} INFO - Job 214: Subtask segunda_tarea
[2023-12-10 19:32:40,989] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-10-10T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:32:41,694] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2022-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-10T00:00:00+00:00
[2023-12-10 19:32:41,703] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:32:41,728] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-12-10 19:32:41,759] {subprocess.py:85} INFO - Output:
[2023-12-10 19:32:41,771] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-12-10 19:32:41,785] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:32:42,158] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20221010T000000, start_date=20231210T193239, end_date=20231210T193242
[2023-12-10 19:32:42,374] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:32:42,686] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
